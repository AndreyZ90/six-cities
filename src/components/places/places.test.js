import React from 'react';
import TestRenderer from 'react-test-renderer';

import Places from '@/components/places/places';

const count = 10;
const childrenNode = <div className="some-children-node">Places children node</div>;

describe(`Places component snapshot`, () => {
  test(`Should correctly render Places component`, () => {
    const tree = TestRenderer.create(
        <Places count={count}>
          {childrenNode}
        </Places>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
