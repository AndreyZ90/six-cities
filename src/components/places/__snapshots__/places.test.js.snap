// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Places component snapshot Should correctly render Places component 1`] = `
<section
  className="cities__places places"
>
  <h2
    className="visually-hidden"
  >
    Places
  </h2>
  <b
    className="places__found"
  >
    10
     places to stay in 
    Amsterdam
  </b>
  <div
    className="some-children-node"
  >
    Places children node
  </div>
</section>
`;
